src/proteomes/components/entry/Overview.tsx

```diff
@@ -0,0 +1,125 @@
+              title: 'Status',
+              content: (
+                <>
+                  <EntryTypeIcon entryType={data.proteomeType} />
+                  {data.proteomeType}
+                </>
+              ),
+            },
+            renderColumnAsInfoListItem(ProteomesColumn.proteinCount),
+            {
+              title: 'Gene Count',
+              content: (
+                <>
+                  {`${data.geneCount}`}
+                  {
+                    // data?.geneCount > 0 && // TODO: API always returns zero (Leo investigating) so removing the condition until this fixed to at least show the link for UX purposes
+                    data.superkingdom && data.taxonomy.taxonId && (
+                      <>
+                        {' '}
+                        <a
+                          href={ftpUrls.referenceProteomes(
+                            data.id,
+                            data.superkingdom,
+                            data.taxonomy.taxonId
+                          )}
+                        >
+                          Download one protein sequence per gene (FASTA)
+                        </a>
+                      </>
+                    )
+                  }
+                </>
+              ),
+            },
+            {
+              title: 'Proteome ID',
+              content: data.id,
+            },
+            (data.taxonomy.taxonId || data.taxonomy.scientificName) && {
+              title: 'Taxonomy',
+              content: <OrganismDataView organism={data.taxonomy} />,
+            },
+            {
+              title: 'Last modified',
+              content: <time>{data.modified}</time>,
+            },
+            data.genomeAssembly?.assemblyId && {
+              title: 'Genome assembly and annotation',
+              content: (
+                <SimpleView
+                  termValue={`${data.genomeAssembly.assemblyId}${
+                    data.genomeAssembly.source
+                      ? ` from ${data.genomeAssembly.source}`
+                      : ''
+                  }`}
+                  linkTo={data.genomeAssembly.genomeAssemblyUrl}
+                />
+              ),
+            },
+            data.genomeAssembly?.level &&
+              renderColumnAsInfoListItem(ProteomesColumn.genomeRepresentation),
+            renderColumnAsInfoListItem(ProteomesColumn.cpd),
+            data.proteomeCompletenessReport?.buscoReport && {
+              title: <BuscoAbbr />,
+              content: (
+                <div className="busco">
+                  <div className="busco__legend">
+                    <BuscoLegend />
+                  </div>
+                  <BuscoView
+                    report={data.proteomeCompletenessReport?.buscoReport}
+                  />
+                </div>
+              ),
+            },
+          ].filter(Boolean) as InfoData
+        }
+      />
+      <div className="description">
+        <hr />
```

Comment:
We might not want to display this if no description is available?

Reply:
Updated